internal/pkg/machine/network/client_linux.go

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//go:build linux
+
+package network
+
+import (
+	ethtoolioctl "github.com/safchain/ethtool"
```

Comment:
The whole reason for the build flags and stuff is to basically do this specific import only on Linux, so that we can build the project on MacOS as well - not to run the emulator itself, it wouldn't work anyway, but unit tests do, and it's handy.